arch.mk

```diff
@@ -23,10 +23,9 @@ arch-setup:
 arch-dokku: arch-setup
 	echo "-> Update package sums, create metadata file and test the build of the package"
 ifeq ($(DOKKU_VERSION),master)
-	git describe --tags > /tmp/VERSION
+	git describe --tags > /tmp/version
 else
-	echo $(DOKKU_VERSION) > /tmp/VERSION
+	echo $(DOKKU_VERSION) > /tmp/version
 endif
-	cat /tmp/VERSION | cut -d '-' -f 1 | cut -d 'v' -f 2 > /tmp/STABLE_VERSION
```

Comment:
We likely still need this `cut` sequence, though not sure how you want to handle it without creating a temp file and overwriting.